pinot-query-runtime/src/test/resources/queries/Comparisons.json

```diff
@@ -390,6 +390,7 @@
     "queries": [
       { "sql": "SELECT val BETWEEN small AND big FROM {tbl}" },
       { "sql": "SELECT val BETWEEN big AND small FROM {tbl}" },
+      { "sql": "SELECT val BETWEEN 2 AND 3 FROM {tbl}" },
```

Comment:
add `BETWEEN 3 AND 2`
literal values actually will be optimized out.

Reply:
done